src/pages/Run/Run.utils.ts

```diff
@@ -23,7 +23,7 @@ export function cleanParametersMap(params: DecodedValueMap<TaskSettingsQueryPara
  * @param viewType Possible viewType string from url
  */
 export function hasViewTypeParam(viewType?: string): viewType is string {
```

Comment:
I can see why you were checking earlier because of the fixed set of expectations.  but is this function still useful?

Reply:
Not very useful. Agreed